nvflare/app_common/wf_comm/__init__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nvflare.app_common.wf_comm.wf_comm_api_spec import WFCommAPISpec
+from nvflare.fuel.data_event.data_bus import DataBus
+
+data_bus = DataBus()
+
+
+def get_wf_comm_api() -> WFCommAPISpec:
+    return data_bus.get_data("wf_comm_api")
```

Comment:
How do we expect users to use this?
In a main.py just call these the same as Client API without writing any type of Classes (Controller / Executor)?

Reply:
Currently has to be used in a class with a `wf_controller_fn_name()` entry function (default `run()`) due to the component builder. Users can either subclass `WFController` which automatically calls this for the user, or they can call it themselves.